Index: regolith-control-center/panels/color/cc-color-device.c
===================================================================
--- regolith-control-center.orig/panels/color/cc-color-device.c
+++ regolith-control-center/panels/color/cc-color-device.c
@@ -158,7 +158,7 @@ cc_color_device_set_expanded (CcColorDev
                               gboolean expanded)
 {
   /* same as before */
-  if (color_device->expanded == expanded)
+  if (color_device == NULL || color_device->expanded == expanded)
     return;
 
   /* refresh */
