Index: regolith-control-center/meson.build
===================================================================
--- regolith-control-center.orig/meson.build
+++ regolith-control-center/meson.build
@@ -1,5 +1,5 @@
 project(
-  'gnome-control-center', 'c',
+  'regolith-control-center', 'c',
         version : '48.1',
         license : 'GPL2+',
   meson_version : '>= 0.58.0'
@@ -356,7 +356,6 @@ subdir('data/icons')
 subdir('po')
 subdir('panels')
 subdir('shell')
-subdir('global-shortcuts-provider')
 subdir('search-provider')
 
 if get_option('tests')
